website/content/docs/agent-and-proxy/agent/template.mdx

```diff
@@ -261,14 +261,15 @@ of the secret's lease duration has elapsed.
 ### Non-Renewable secrets
 
 If a secret or token isn't renewable or leased, Vault Agent will fetch the secret every 5 minutes.
-This can be configured using Template config [static_secret_render_interval](/vault/docs/agent-and-proxy/agent/template#static_secret_render_interval) (requires Vault 1.8+).
+This can be configured using the `template_config` stanza value [static_secret_render_interval](/vault/docs/agent-and-proxy/agent/template#static_secret_render_interval) (requires Vault 1.8+).
 Non-renewable secrets include (but not limited to) [KV Version 2](/vault/docs/secrets/kv/kv-v2).
 
 ### Non-Renewable leased secrets
 
-If a secret or token is non-renewable but leased, Vault Agent will fetch the secret when 85% of the
-secrets time-to-live (TTL) is reached. Leased, non-renewable secrets include (but not limited
+If a secret or token is non-renewable but leased, Vault Agent will fetch the secret when roughly
```

Comment:
"roughly 90%" isn't an actionable measure. Is there a reason we can't just say "90%"?

Reply:
Yeah, there's some jitter to avoid thundering herd-esque problems. This was the case before, too, and we've had complaints that it wasn't _exactly_ 85%, so I wanted to guard against that. You're right though, I should be more explicit. Let me update!

Reply:
The intent is 3% jitter either way, but there's a bug that's making it slightly less accurate in that, so I was also trying to hedge against that too.